### PR TITLE
{CI} Disable CodeQL task

### PR DESCRIPTION
**Pull request by bebound**
CodeQL tasks always timeout in these tasks. https://dev.azure.com/azclitools/public/_build/results?buildId=172894&view=logs&jobId=61bf4700-6d0f-55d3-0cc1-f8d3dd51c404
image
image

Usually, `Finalize CodeQL (auto-injected)` takes about 30min. I have no idea why they fail in these 6 tasks.

Luckily, they are all in `azure-pipelines.yml`. Even though we disable it here, **the CodeQL task still runs in `azure-pipelines-full-tests.yml`**. We still meet the security requirements.

PS: CodeQL also make some tasks in PowerShell fail: https://github.com/Azure/azure-powershell/pull/25560

Ref:
https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/configuring-codeql3000-ado-pipelines
https://portal.microsofticm.com/imp/v5/incidents/details/520675856/summary

### CODE REVIEW

**Comment by yonzhan** [comment]
CI

**Review by Pan-Qi — APPROVED** [review]

**Comment by jiasli** [comment]
Will CodeQL tasks block this PR too? Shall we force merge it or wait for a lucky pass?

**Review by jiasli — APPROVED** [review]

File: azure-pipelines.yml
Comment (bebound) [inline azure-pipelines.yml:18]: I have to use this syntax instead of `Codeql.Enabled: false`, because we use varable `template`.

Ref: https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#specify-variables

**Comment by bebound** [comment]
> Will CodeQL tasks block this PR too? Shall we force merge it or wait for a lucky pass?

CodeQL only runs after the PR is merged into the dev branch.

**Review by jiasli — APPROVED** [review]

**Pull request merged**
